app/code/Magento/Review/Model/Review/AddRatingVotesToCustomerReviews.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Review\Model\Review;
+
+use Magento\Review\Model\ResourceModel\Rating\Option\Vote\Collection as OptionVoteCollection;
+use Magento\Review\Model\ResourceModel\Rating\Option\Vote\CollectionFactory as OptionVoteCollectionFactory;
+use Magento\Review\Model\ResourceModel\Review\Product\Collection;
+
+/**
+ * The model that adds the rating votes to reviews
+ */
+class AddRatingVotesToCustomerReviews
```

Comment:
Since we are not introducing new API please move those classes from Review module to ReviewGraphQl module.